bin/docker-entrypoint

```diff
@@ -23,9 +23,15 @@ dev_scheduler() {
 }
 
 worker() {
-  echo "Starting RQ worker..."
+  WORKERS_COUNT=${WORKERS_COUNT:-2}
+  echo "Starting $WORKERS_COUNT RQ workers for queues $QUEUES..."
+
+  for i in `seq 1 $WORKERS_COUNT`;
+  do
+    exec /app/manage.py rq worker $QUEUES &
+  done    
 
-  exec /app/manage.py rq worker $QUEUES
+  sleep infinity
```

Comment:
I wonder if it will be more robust/cleaner to use something like [Honcho](https://github.com/nickstenning/honcho). This way one worker exiting/crashing won't kill the others.

And to clarify: I don't mean to use the Honcho CLI, but rather use its `Manager` and `Process` classes from our code directly, skipping the need to define create a `Procfile`. We can also implement our own logic, but I guess Honcho took care of many of the possible issues with managing multiple processes.

Reply:
+1 for Honcho!

Reply:
Using Honcho directly in 0d95d24. I like it because it will kill all workers and exit if one of the worker processes goes AWOL, but it feels a tad strange to specify a command to run within the code 🤷‍♂

Reply:
> it feels a tad strange to specify a command to run within the code

I agree. I sort of expected that we can provide a function to invoke as the subprocess, but it makes sense that their code assumes you provide them an executable path.

> I like it because it will kill all workers and exit if one of the worker processes goes AWOL

Better option would be to just restart it, no?

(btw, you need to rebase :grimacing:)

Reply:
> Better option would be to just restart it, no?
Yeah, it does that now.

> (btw, you need to rebase 😬)
That. Was. Fun.

Reply:
Now it restarts all of them instead of only the faulty one. Between the previous version (process exits when one or more workers misbehave) and current one (all workers restart in a loop when one of the workers misbehave), I think the safer one is the first.

While the end result is somewhat the same, it leaves more room for seeing what's going on. For example with ECS, we will see tasks restarting frequently.

Reply:
If we want to restart _only_ the faulty workers, we probably need to move towards something like supervisor (which was what we initially had in mind, but the purpose of this PR was to provide something lighter 😉)

Reply:
Anyway, reverted the auto-restart for now.